test/post-911-gib-status/components/UserInfoSection.unit.spec.js

```diff
@@ -1,19 +1,52 @@
 import React from 'react';
 import SkinDeep from 'skin-deep';
 import { expect } from 'chai';
+import _ from 'lodash';
 
 import UserInfoSection from '../../../src/js/post-911-gib-status/components/UserInfoSection.jsx';
 
 const props = {
   enrollmentData: {
   }
 };
+const currentHeadingSelector = '#up-to-date-header';
 
 describe('<UserInfoSection>', () => {
   it('should render', () => {
     const tree = SkinDeep.shallowRender(<UserInfoSection {...props}/>);
     const vdom = tree.getRenderOutput();
     expect(vdom).to.not.be.undefined;
   });
-  // maybe test print button does something
+
+  describe('props.showCurrentAsOfAlert=false', () => {
+    it('should omit the "current as of" date', () => {
+      const currentAsOfProps = _.merge({}, props, { showCurrentAsOfAlert: false });
+      const tree = SkinDeep.shallowRender(<UserInfoSection {...currentAsOfProps}/>);
+      expect(tree.subTree(currentHeadingSelector)).to.be.false;
+    });
+  });
+
+  describe('props.showCurrentAsOfAlert=null', () => {
+    it('should omit the "current as of" date', () => {
+      const tree = SkinDeep.shallowRender(<UserInfoSection {...props}/>);
+      expect(tree.subTree(currentHeadingSelector)).to.be.false;
+    });
+  });
+
+  describe('props.showCurrentAsOfAlert=true', () => {
+    it('should display the "current as of" date', () => {
+      const currentAsOfProps = _.merge({}, props, { showCurrentAsOfAlert: true });
+      const tree = SkinDeep.shallowRender(<UserInfoSection {...currentAsOfProps}/>);
+      expect(tree.subTree(currentHeadingSelector)).to.not.be.false;
+    });
+  });
+
+  describe('props.percentageBenefit=null', () => {
```

Comment:
Can you add a TODO here that we need to figure out how to handle corrupt data and mention this issue? https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3336
We might call this group of tests something more general like
```
describe('enrollment data is incomplete', () => {
```
in case we want to add other tests